junebug/channel.py

```diff
@@ -201,6 +201,19 @@ def get_all(cls, redis):
         channels = yield redis.smembers('channels')
         returnValue(channels)
 
+    @classmethod
+    @inlineCallbacks
+    def start_all_channels(cls, redis, config, parent):
+        '''Ensures that all of the stored channels are running'''
+        for id in (yield cls.get_all(redis)):
+            try:
+                yield cls.from_id(redis, config, id, parent)
```

Comment:
I think `channel.start` is a no-op for started channels (or should be), so maybe we should just have the `channel.start` apply to both cases (both in memory or created from redis).